lib/modules/datasource/metadata-manual.ts

```diff
@@ -59,6 +81,26 @@ export const manualChangelogUrls: Record<string, Record<string, string>> = {
 // Use this object to define manual source URLs for packages
 // Only necessary if the datasource is unable to locate the source URL itself
 export const manualSourceUrls: Record<string, Record<string, string>> = {
+  maven: {
```

Comment:
can this reverted? the package should have correct source urls now

Reply:
not yet released.